homeassistant/components/dynalite/bridge.py

```diff
@@ -54,9 +54,9 @@ def update_signal(self, device: "DynaliteBaseDevice" = None) -> str:
         return signal
 
     @callback
-    def update_device(self, device: "DynaliteBaseDevice") -> None:
+    def update_device(self, device: "DynaliteBaseDevice" = None) -> None:
```

Comment:
This type should be optional since it can be none:

```suggestion
    def update_device(self, device: Optional["DynaliteBaseDevice"] = None) -> None:
```

```python
from typing import Optional
```

Reply:
done